source/framework/tools/inc/TRestStringHelper.h

```diff
@@ -36,7 +36,7 @@ Double_t StringToDouble(std::string in);
 Int_t StringToInteger(std::string in);
 std::string IntegerToString(Int_t n, std::string format = "%d");
 std::string DoubleToString(Double_t d, std::string format = "%4.2lf");
-Bool_t StringToBool(std::string in);
+Bool_t StringToBool(const std::string& in);
```

Comment:
The compilation failure is due to this `const` keyword. It makes the argument unmatched. If we think this change is necessary then we shall open a new PR to replace all the `AddConverter(xxx)` with const-input methods.

Reply:
I think it should be implemented if it's not too much work, its not high priority but using `const T&` should always be used to pass non built-in types or atleast be the default option, so I think we should support it.